components/Dropdown/Dropdown.less

```diff
@@ -14,6 +14,8 @@
     }
   }
   .dropdown-content {
+    z-index: -10;
```

Comment:
why do you have Z-index -10 ? take a look at documentation - https://www.w3schools.com/cssref/pr_pos_z-index.asp